ui/apps/dashboard/src/components/Layout/Layout.tsx

```diff
@@ -7,21 +7,28 @@ import { getNavCollapsed } from '@/app/actions';
 import { BillingBanner } from '@/components/BillingBanner';
 import { getProfileDisplay } from '@/queries/server-only/profile';
 import type { Environment } from '@/utils/environments';
+import { getBooleanFlag } from '../FeatureFlags/ServerFeatureFlag';
 import SideBar from './SideBar';
 
 type LayoutProps = {
-  envSlug?: string;
   activeEnv?: Environment;
   children: ReactNode;
 };
 
 export default async function Layout({ activeEnv, children }: LayoutProps) {
   const collapsed = await getNavCollapsed();
   const profile = await getProfileDisplay();
+  const enableQuickSearchV2 = await getBooleanFlag('quick-search-v2');
 
   return (
     <div className="fixed z-50 flex h-screen w-full flex-row justify-start overflow-y-scroll overscroll-y-none">
-      <SideBar collapsed={collapsed} activeEnv={activeEnv} profile={profile} />
+      <SideBar
+        activeEnv={activeEnv}
+        collapsed={collapsed}
+        enableQuickSearchV2={enableQuickSearchV2}
+        envSlug={activeEnv?.slug ?? 'production'}
```

Comment:
Do we need to pass this here? Why not just using the activeEnv.slug inside the Sidebar instead?

Reply:
Good point -- done